platforms/ios/demo/src/MapViewController.m

```diff
@@ -18,6 +18,15 @@ - (void)addAlert:(NSString *)message withTitle:(NSString *)title;
 
 @implementation MapViewControllerDelegate
 
+- (void)mapView:(TGMapViewController *)mapView didCompleteSceneUpdatesWithErrors:(NSArray<TGSceneUpdateStatus *> *)updateStatuses
+{
+    for (TGSceneUpdateStatus* updateStatus in updateStatuses) {
+        NSLog(@"Scene update error for update with path %@ and value %@",
```

Comment:
Yup, fine to have logs in the demo app. But my question was more on the usage of NSError :D. Will be good to have usage of the NSError from @msmollin though. Thanks